### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by Sea-n**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Improve Repo

### Description 
Use "Git**H**ub" instead of "Github"
Use "You**T**ube" instead of "Youtube"

### Why is this valuable (or not)?
Improve readability

### How do we know it's really free?
:cat2:

### Checklist:
- [x] Not a duplicate
- [ ] Included author(s) if appropriate
- [ ] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-books.md [hunk 1]
```diff
@@ -1386,9 +1386,9 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Git Reference](https://web.archive.org/web/20170602211147/http://gitref.org/)
 * [Git Succinctly, Syncfusion](https://www.syncfusion.com/resources/techportal/ebooks/git) (PDF, Kindle) (email address *requested*, not required)
 * [Git Workflows](http://documentup.com/skwp/git-workflows-book)
-* [How to Collaborate on Github](https://github.com/eonist/How-to-collaborate-on-github) - André J
-* [Intoduction to Git and Github](https://launchschool.com/books/git) - Launch School
-* [Intoduction to Git and Github - Tutorial](http://cse.unl.edu/~cbourke/gitTutorial.pdf) - Dr. Chris Bourke (PDF)
+* [How to Collaborate on GitHub](https://github.com/eonist/How-to-collaborate-on-github) - André J
+* [Intoduction to Git and GitHub](https://launchschool.com/books/git) - Launch School
+* [Intoduction to Git and GitHub - Tutorial](http://cse.unl.edu/~cbourke/gitTutorial.pdf) - Dr. Chris Bourke (PDF)
```
Comment (SethiPandi) [inline free-programming-books.md:1391]: It seems to be the title of this resource doesn't even contain GitHub, it's only "Introduction to Git", could we update it?

```suggestion
* [Intoduction to Git](http://cse.unl.edu/~cbourke/gitTutorial.pdf) - Dr. Chris Bourke (PDF)
```

Comment (Sea-n) [reply free-programming-books.md:1391]: Thanks! And the mis-spelling "Int**r**oduction" also fixed.

Comment (SethiPandi) [reply free-programming-books.md]: Didn't even notice that! Nice catch! ^-^'

**Comment by Sea-n** [comment]
Hello @eshellman, could I ask why you closed this PR?

CC @SethiPandi

**Comment by eshellman** [comment]
Because I didn't have time to check through the whole thing, and you didn't respect the source's capitalization on the one item I checked. Feel free to reopen, we won't be as grumpy later in November. In addition, PR's that change many pages cause many merge conflicts, which make it much more difficult to clean up our large backlog. I'd recommend separating this into 3 PRs, one addressing our capitalization of YouTube, one addressing GitHub, and another address source mismatch. Remember also that in capitalization may normalize differently depending on localization - probably not, but please check. Also, please read my comments in  #4579: "We can elide 'YouTube' and 'Github" from listings. " Maybe I should have written "We should elide 'YouTube' and 'GitHub" from listings."

**Pull request closed without merging**